src/main/java/org/dcsa/core/repository/RowMapper.java

```diff
@@ -0,0 +1,131 @@
+package org.dcsa.core.repository;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.introspect.Annotated;
+import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import io.r2dbc.spi.ColumnMetadata;
+import io.r2dbc.spi.Row;
+import io.r2dbc.spi.RowMetadata;
+import org.dcsa.core.exception.DatabaseException;
+import org.dcsa.core.extendedrequest.QueryField;
+import org.dcsa.core.query.DBEntityAnalysis;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+
+public class RowMapper {
+    private static final Integer DATABASE_INTERVAL_NATIVE_TYPE = 1186;
+
+    public static <T> T mapRow(Row row, RowMetadata metadata, DBEntityAnalysis<T> dbEntityAnalysis, Class<T> modelClass, boolean ignoreUnknownProperties) {
```

Comment:
I think we should make this "non-static" and initialize the object mapper as field.  Something like:

```java
    private final ObjectMapper objectMapper = new ObjectMapper()
                   .registerModule(new JavaTimeModule()) // Needed to process time objects.
                   .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
                   .disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE)
                   .enable(SerializationFeature.WRITE_DATES_WITH_ZONE_ID)
                   .setAnnotationIntrospector(new JacksonAnnotationIntrospector() {
                      // Insert body here - too lazy to copy-waste it as multiline copy in GH embeds HTML fragments ...
                   });
```